Add wildcard query documentation (#10198)
DataDog/documentation: content/en/network_monitoring/performance/network_page.md

Once the custom facet is created, use this tag to filter and aggregate traffic in the network page and map. All custom facets can be viewed in the bottom `Custom` section of the facet panels.

### Wildcard search

> To perform a multi-character wildcard search, use the * symbol as follows:

Reviewer: ```suggestion
To perform a multi-character wildcard search, use the `*` symbol as follows:
```

- `service:web*` matches all services that start with web
- `service:*web` matches all services that end with web
- `service:*web*` matches all services that contain the string web

Wildcard searches work within facets with this syntax. This query returns all the services that end with the string mongo:

`service:*mongo`

> To learn more, see our documentation on [search syntax][1].

Reviewer: ```suggestion
To learn more, see the [search syntax][1] documentation.
```

- To rename a view: hover over a view in the Saved Views list and click the gear icon to *Edit name*.
- To share a view: hover over a view in the Saved Views list and click the link icon to *Copy permalink*.

> To learn more, see our documentation on [Saved Views][2].

Reviewer: ```suggestion
To learn more, see the [Saved Views][2] documentation.
```
